.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,63 @@
+---
+name: Bug report
+about: Create a report to help us reproduce and correct the bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+#### Describe the bug
+A clear and concise description of what the bug is.
+
+#### Steps/Code to Reproduce
+Please add a minimal example that we can reproduce the error by running the
+code. Be as succint as possible, do not depend on external data. In short, we
+are going to copy-paste your code in `IPython` and we expect to get the same
```

Comment:
```suggestion
are going to copy-paste your code and we expect to get the same
```

In case they want to output the `[1]: ...` from Ipython...

Reply:
We could still run those in IPython :) but true.